src/activity-logger/background/log-page-visit.js

```diff
@@ -74,3 +74,18 @@ export async function maybeLogPageVisit({ tabId, url }) {
         url,
     })
 }
+
+export async function logPageVisitForBookmark({ tabId, url }) {
```

Comment:
This does not seem necessary. It is just doing the first half of `logPageVisit`, finishing after the page is stored. I think maybe ignore this here and do the page creation in `createBookmarkByTab`.